packages/in_app_purchase/example/lib/main.dart

```diff
@@ -0,0 +1,360 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:in_app_purchase/in_app_purchase.dart';
+import 'package:in_app_purchase_platform_interface/in_app_purchase_platform_interface.dart';
+import 'package:in_app_purchase_tizen/billing_manager_wrappers.dart';
+import 'package:in_app_purchase_tizen/in_app_purchase_tizen.dart';
+
+void main() {
+  WidgetsFlutterBinding.ensureInitialized();
+
+  InAppPurchaseTizenPlatform.registerPlatform();
+  runApp(_MyApp());
+}
+
+// To try without auto-consume, change `true` to `false` here.
+const bool _kAutoConsume = true;
+
+const String _kAppId = '3201504002021';
+const String _kServerType = 'DEV';
+const String _kCountryCode = 'US';
+const int _kPageSize = 20;
+const int _kPageNum = 1;
+const String _kSecurityKey = 'YxE757K+aDWHJXa0QMnL5AJmItefoEizvv8L7WPJAMs=';
```

Comment:
What kind of key is this? Is it an officially provided test key that can be used for testing purposes, or is it someone's personal security key? If it is someone's personal key, should we only indicate the need to input the security as a comment?

Reply:
> Is it an officially provided test key that can be used for testing purposes, or is it someone's personal security key?

`_kSecurityKey` is provided by users, it is a personal key. As the [documentation ](https://developer.samsung.com/smarttv/develop/guides/samsung-checkout/implementing-the-purchase-process.html#Generating-Check-Values)said: 
The DPI security key is used only by Samsung Smart TV applications for open API calls. Do not expose this key. You can use a key management server for greater security.
Users can retrieve the key on their own.

Reply:
Fixed.